Sources/Rendering/Core/Prop3D/index.js

```diff
@@ -81,6 +84,15 @@ function vtkProp3D(publicAPI, model) {
     publicAPI.modified();
   };
 
+  publicAPI.rotateQuaternion = (orientationQuaternion) => {
+    const oriQuatMat = mat4.fromQuat(
+      new Float64Array(16),
+      orientationQuaternion
+    );
+    mat4.multiply(model.rotation, model.rotation, oriQuatMat);
+    publicAPI.modified();
```

Comment:
if `orientationQuaternion` is identity, it shouldn't call `modified`

Reply:
see latest commits